runtime/parachains/src/inclusion_inherent.rs

```diff
@@ -70,13 +68,11 @@ decl_module! {
 		}
 
 		fn on_finalize() {
-			if Included::take().is_none() {
-				panic!("Bitfields and heads must be included every block");
-			}
+			Included::take();
```

Comment:
Is there a useful way to emit a log line here? Even if we don't want to panic anymore, it might be worth mentioning to the node operator.